src/PrestaShopBundle/Command/UpdateSchemaCommand.php

```diff
@@ -0,0 +1,211 @@
+                    foreach ($matches[2] as $matchKey => $fieldName) {
+                        // remove table name
+                        $matches[0][$matchKey] = preg_replace('/(.+?) CHANGE/',
+                            ' CHANGE', $matches[0][$matchKey]);
+                        // remove quote
+                        $originalFieldName = $fieldName;
+                        $fieldName = str_replace('`', '', $fieldName);
+                        // get old default value
+                        $query = $conn->query('SHOW FULL COLUMNS FROM '.$tableName.' WHERE Field="'.$fieldName.'"');
+                        $results = $query->fetchAll();
+                        $oldDefaultValue = $results[0]['Default'];
+                        $extra = $results[0]['Extra'];
+                        if ($oldDefaultValue !== null
+                            && strpos($oldDefaultValue, 'CURRENT_TIMESTAMP') === false) {
+                            $oldDefaultValue = "'".$oldDefaultValue."'";
+                        }
+                        if ($oldDefaultValue === null) {
+                            $oldDefaultValue = 'NULL';
+                        }
+                        // set the old default value
+                        if (!($results[0]['Null'] == 'NO' && $results[0]['Default'] === null)
+                            && !($oldDefaultValue === 'NULL'
+                                && strpos($matches[0][$matchKey], 'NOT NULL') !== false)
+                            && (strpos($matches[0][$matchKey], 'BLOB') === false)
+                            && (strpos($matches[0][$matchKey], 'TEXT') === false)
+                        ) {
+                            if (preg_match('/DEFAULT/', $matches[0][$matchKey])) {
+                                $matches[0][$matchKey] =
+                                    preg_replace('/DEFAULT (.+?)(,|$)/', 'DEFAULT '.
+                                        $oldDefaultValue.'$2'.' '.$extra, $matches[0][$matchKey]);
+                            } else {
+                                $matches[0][$matchKey] =
+                                    preg_replace('/(.+?)(,|$)/uis', '$1 DEFAULT '.
+                                        $oldDefaultValue.' '.$extra.'$2', $matches[0][$matchKey]);
+                            }
+                        }
+                        $updateSchemaSql[$key] = preg_replace('/ CHANGE '.$originalFieldName.' (.+?)(,|$)/uis',
+                            $matches[0][$matchKey], $updateSchemaSql[$key]);
+                    }
+                }
+            }
+        }
+
+        $sqls += count($updateSchemaSql);
+        // Now execute the queries!
+        foreach ($updateSchemaSql as $sql) {
+            try {
+                $output->writeln('Executing: ' . $sql);
+                $conn->executeQuery($sql);
+            } catch (\Exception $e) {
+                $conn->rollBack();
+                throw($e);
```

Comment:
You should perhaps write the exception message in $output instead of throwing the exception?